src/components/POLine/Eresources/EresourcesForm.js

```diff
@@ -90,6 +97,25 @@ class EresourcesForm extends Component {
             name="eresource.createInventory"
           />
         </Col>
+        <Col xs={6} md={3}>
+          <FormattedMessage id="ui-orders.dropdown.select">
+            {(placeholder) => (
+              <Field
+                component={Select}
+                dataOptions={[{ label: placeholder, value: '' }, ...materialTypes]}
+                fullWidth
+                label={<FormattedMessage id="ui-orders.poLine.materialType" />}
+                name="eresource.materialType"
+                normalize={(value) => {
```

Comment:
I think we might need this as a common normalizer for almost every optional select....